flask_monitor/requirements.txt

```diff
@@ -0,0 +1,43 @@
+appdirs==1.4.3
+atomicwrites==1.3.0
+attrs==19.3.0
+black==19.10b0
+certifi==2019.11.28
+chardet==3.0.4
+Click==7.0
+codecov==2.0.15
+coverage==5.0.3
+dparse==0.4.1
+entrypoints==0.3
+flake8==3.7.9
+Flask==1.1.1
+idna==2.8
+isort==4.3.21
+itsdangerous==1.1.0
+Jinja2==2.11.0
+MarkupSafe==1.1.1
+mccabe==0.6.1
+more-itertools==8.1.0
+mypy==0.740
+mypy-extensions==0.4.3
+packaging==20.1
+pathspec==0.7.0
+pluggy==0.13.1
+prometheus-client==0.7.1
+py==1.8.1
+pycodestyle==2.5.0
+pyflakes==2.1.1
+pyparsing==2.4.6
+pytest==5.2.2
+pytest-cov==2.8.1
+PyYAML==5.3
+regex==2020.1.8
+requests==2.22.0
+safety==1.8.5
+six==1.14.0
+toml==0.10.0
+typed-ast==1.4.1
+typing-extensions==3.7.4.1
+urllib3==1.25.8
+wcwidth==0.1.8
+Werkzeug==0.16.1
```

Comment:
@gutorc92, do we really need all these requirements?

Reply:
Requirements file was generated by pipenv. Pipenv write down all packages used on the project.